docs/generate-docs.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+## Set up documentation so README.md is reflected onto apollographql.com/docs/android
+
+set -e
+
+echo -e "---\ntitle: Get started\ndescription: Learn how to quickly set up Apollo Client in your Android app\n---\n\n$(cat ../README.md | grep -v 'the official docs')" > source/essentials/get-started.md
```

Comment:
One question: are the URL links ok? I'm not 100 sure but there is a slight difference between how links to other docs work between README.md and others. Either we would need to convert those links or keep the compatible.

Reply:
> One question: are the URL links ok?

I think they are. Gatsby actually has a check that fails if they're not ok.

Most of the url in the README.md are absolute, which explains why they work. There's only one that's not absolute and it's an anchor: `[downloading a schema file](#downloading-a-schemajson-file).`

Reply:
Anchors work fine. And you're right, gatsby verify links. I think we should be ok, then. We cannot put relative links to code in the repo, we will need to put full links.

Reply:
Should we merge this then?

Reply:
LGTM! I made a rebase there: https://github.com/apollographql/apollo-android/pull/2053